assets/src/edit-story/components/header/buttons/update.js

```diff
@@ -72,21 +85,25 @@ function Update() {
     default:
       text = __('Save draft', 'web-stories');
       return (
-        <Outline
-          onClick={() => saveStory({ status: 'draft' })}
-          isDisabled={
-            !hasMetaBoxes && (isSaving || isUploading || !hasNewChanges)
-          }
-        >
-          {text}
-        </Outline>
+        <Tooltip title={text} placement={TOOLTIP_PLACEMENT.BOTTOM} hasTail>
```

Comment:
Should we add the shortcut `mod+s` here as well?

Reply:
`useGlobalKeyDownEffect` already exists in this file for `mod+s`, see a bit above (unchanged part of the file). If that's what you meant.

Reply:
I meant to show the actual shortcut in the tooltip itself by adding `shortcut="mod+s"` here.